cockpit/src/main/java/com/polidea/cockpit/paramsedition/CockpitDialogPresenter.kt

```diff
@@ -36,8 +33,19 @@ class ParamsEditionPresenter(private val view: ParamsEditionContract.View) : Par
         view.reloadAll()
     }
 
+    override fun <T : Any> onParamChange(position: Int, newValue: T) {
+        model.setValue(position, newValue)
+    }
+
+    override fun expand() {
+        view.expand()
+    }
+
+    override fun collapse() {
+        view.collapse()
+    }
+
```

Comment:
`CockpitDialogPresenter.save()` method seems to be unused . Model is always saved after `stop()` method call.